### ISSUE TITLE
Per address statistics

### ISSUE THREAD

**Issue by RollaTroll** [micz/ThunderStats#83]
Graph statistics sorted by address.
Graph average email body length (by words and letters). Include sorting by average length and longest/shortest length. Allow custom averaging period.
Graph average messages per day.
If possible, allow truncation of signatures from messages before averaging body length.
Text from previous message (if a reply) should ignored

**Comment by micz** [comment]
Are you thinking about a new tab with a field to write the sender/recipient address?

**Comment by RollaTroll** [comment]
Yes, that would be ideal.

**Comment by guerrero4de** [comment]
Yes that would be very useful (even without bodylength, only messaes per day)
